Description: Fix incorrect return type for katedesc_restart
 The function is defined to return void in kate_lexer.c,
 and its caller should not pretend it return an int.
Author: Petter Reinholdtsen <pere@hungry.com>
Bug-Debian: http://bugs.debian.org/749444
Forwarded: no
Reviewed-By: Petter Reinholdtsen <pere@hungry.com>
Last-Update: 2014-10-22

--- libkate-0.4.1.orig/tools/kateenc.c
+++ libkate-0.4.1/tools/kateenc.c
@@ -36,7 +36,7 @@
 #include "katedesc.h"
 
 extern int katedesc_parse(void);
-extern int katedesc_restart(FILE*);
+extern void katedesc_restart(FILE*);
 extern FILE *katedesc_in;
 extern FILE *katedesc_out;
 extern int nerrors;
